### PR TITLE
convert usdm to wusdm

### PR DESCRIPTION
**Pull request by pete-ipor**

### CODE REVIEW

File: test/amm-wusdm/IUSDM.sol
Comment (mario-ipor) [inline test/amm-wusdm/IUSDM.sol:1]: Change IUSDM.sol to IWUSDM.sol

File: test/amm-wusdm/ProvideUsdm.t.sol
Comment (mario-ipor) [inline test/amm-wusdm/ProvideUsdm.t.sol:1]: Change file name ProvideUsdm.t.sol to ProvideWusdm.t.sol

File: test/amm-wusdm/UsdmForkAmmGovernanceServiceTest.t.sol
Comment (mario-ipor) [inline test/amm-wusdm/UsdmForkAmmGovernanceServiceTest.t.sol:1]: Change file name to WUsdmFork...

File: test/amm-wusdm/UsdmForkAmmUsdmExchangeRate.t.sol
Comment (mario-ipor) [inline test/amm-wusdm/UsdmForkAmmUsdmExchangeRate.t.sol:1]: Change file name

File: test/amm-wusdm/UsdmForkAmmUsdmExchangeRate.t.sol [hunk 1]
```diff
@@ -8,22 +8,22 @@ import "../../contracts/interfaces/types/AmmTypes.sol";
 
 contract UsdmForkAmmWstEthExchangeRateTest is UsdmTestForkCommon {
```
Comment (mario-ipor) [inline test/amm-wusdm/UsdmForkAmmUsdmExchangeRate.t.sol:9]: Change contract name to WUsdm...

File: test/amm-wusdm/UsdmTestForkCommon.sol [hunk 2]
```diff
@@ -151,8 +159,15 @@ contract UsdmTestForkCommon is Test {
...
+
+        vm.prank(HolderUsdm);
+        IUSDM(USDM).approve(WUSDM, 100e18);
+
+        vm.prank(HolderUsdm);
+        ERC4626Upgradeable(WUSDM).deposit(100e18, HolderUsdm);
+
         vm.prank(HolderUsdm);
-        IAmmPoolsServiceUsdm(IporProtocolRouterProxy).provideLiquidityUsdmToAmmPoolUsdm(HolderUsdm, 10e18);
+        IAmmPoolsServiceWusdm(IporProtocolRouterProxy).provideLiquidityWusdmToAmmPoolWusdm(HolderUsdm, 10e18);
```
Comment (mario-ipor) [inline test/amm-wusdm/UsdmTestForkCommon.sol:171]: holder usdm or wusdm?

**Review by mario-ipor — CHANGES_REQUESTED** [review]

File: test/amm-wusdm/IUSDM.sol
Comment (pete-ipor) [reply test/amm-wusdm/IUSDM.sol:1]: no, it is interface for usdm not wusdm

File: test/amm-wusdm/UsdmTestForkCommon.sol (hunk 2 above)
Comment (pete-ipor) [reply test/amm-wusdm/UsdmTestForkCommon.sol:171]: holders of usdm

**Review by mario-ipor — APPROVED** [review]

**Pull request merged**